tests/Microsoft.Bot.Builder.Dialogs.Declarative.Tests/JsonLoadTests.cs

```diff
@@ -327,96 +313,96 @@ await BuildTestFlow("HttpRequest.main.dialog")
             .StartTestAsync();
         }
 
-        [TestMethod]
+        [Fact]
         public async Task JsonDialogLoad_QnAMakerDialog_ActiveLearning_WithProperResponse()
         {
             var suggestionList = new List<string> { "Q1", "Q2", "Q3" };
             var suggestionActivity = QnACardBuilder.GetSuggestionsCard(suggestionList, "Did you mean:", "None of the above.");
             var qnAMakerCardEqualityComparer = new QnAMakerCardEqualityComparer();
 
-            await BuildQnAMakerTestFlow()
+            await BuildQnAMakerTestFlow(nameof(JsonDialogLoad_QnAMakerDialog_ActiveLearning_WithProperResponse))
             .Send("Q11")
                 .AssertReply(suggestionActivity, equalityComparer: qnAMakerCardEqualityComparer)
             .Send("Q1")
                 .AssertReply("A1")
             .StartTestAsync();
         }
 
-        [TestMethod]
+        [Fact]
         public async Task JsonDialogLoad_QnAMakerDialog_ActiveLearning_WithNoResponse()
         {
             var suggestionList = new List<string> { "Q1", "Q2", "Q3" };
             var suggestionActivity = QnACardBuilder.GetSuggestionsCard(suggestionList, "Did you mean:", "None of the above.");
             var qnAMakerCardEqualityComparer = new QnAMakerCardEqualityComparer();
-            var noAnswerActivity = "Answers not found in kb.";
+            const string noAnswerActivity = "Answers not found in kb.";
```

Comment:
When the assigned object is explicit shouldn't go var? or this changes when its a const?

Reply:
Var is for variables, if the value doesn't change it must be a const.